website/content/docs/enterprise/namespaces/index.mdx

```diff
@@ -0,0 +1,162 @@
+---
+layout: docs
+page_title: Namespaces - Vault Enterprise
+description: >-
+  Vault Enterprise has support for Namespaces, a feature to enable Secure
+  Multi-tenancy (SMT) and self-management.
+---
+
+# Vault Enterprise namespaces <EnterpriseAlert product=vault inline=true />
+
+Many organizations implement Vault as a service to provide centralized
+management of sensitive data and ensure that the different teams in an
+organization operate within isolated environments known as **tenants**.
+
+Multi-tenant environments have the following implementation challenges:
+
+- **Tenant isolation**. Teams within a Visualization as a Service (VaaS)
+  environment require strong isolation for their policies, secrets, and
+  identities. Tenant isolation may also be required due to organizational
+  security and privacy requirements or to address compliance regulations like
+  [GDPR](https://gdpr.eu).
+- **Long-term management**. Tenants typically have different policies and teams
+  request changes to their tenants at different rates. As a result, managing a
+  multi-tenant environment can become difficult for a single team as the number
+  of tenants within the organization grows.
+
+Namespaces support secure multi-tenancy (**SMT**) within a single Vault
+Enterprise instance with tenant isolation and administration delegation so Vault
+administrators can empower delegates to manage their own tenant environment.
+
+When you create a namespace, you establish an isolated environment with separate
+login paths that functions as a mini-Vault instance within your Vault
+installation. Users can then create and manage their sensitive data within the
+confines of that namespace, including:
+
+- secret engines
```

Comment:
I just want to note a historical point about inconsistent usage on this term across both documentation and tutorials which I hope we can standardize on. Historically, the term used most for this is **secrets engine**, with an emphasis on the plural form of secret, since an engine can deal with a multitude of secrets.

That said, in documentation there are about 1075 uses of `secrets engine` and  about 70 uses of `secret engine`.

With tutorials, it is similar: ~1436 uses of `secrets engine` and ~53 uses of `secret engine`.

I think we should standardize on preferring the correct term, **secrets engine** and update those instances of the incorrect term "secret engine".

We can discuss this further on the team call, but my plan is to fix the 53 incorrect usages in our tutorials.

Reply:
Agreed we need to standardize. TBH, I'm not convinced that "secrets engine" **or** "secret engine" are very helpful terms =\